desc/equilibrium/coords.py

```diff
@@ -312,6 +322,103 @@ def fixup(x, *args):
     return out
 
 
+def map_clebsch_coords(
+    clebsch_coords,
+    iota,
+    L_lmn,
+    L_basis,
+    tol=1e-6,
+    maxiter=20,
+    full_output=False,
+    **kwargs,
+):
+    """Find θ for given Clebsch field line poloidal label α.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium to use.
+    clebsch_coords : ndarray
+        Shape (k, 3).
+        Clebsch field line coordinates [ρ, α, ζ]. Assumes ζ = ϕ.
+        Each row is a different point in space.
+    iota : ndarray
+        Shape (k, )
+        Rotational transform on each node.
+    L_lmn : ndarray
+        Spectral coefficients for lambda.
+    L_basis : Basis
+        Spectral basis for lambda.
+    tol : float
+        Stopping tolerance.
+    maxiter : int > 0
+        Maximum number of Newton iterations.
+    full_output : bool, optional
+        If True, also return a tuple where the first element is the residual from
+        the root finding and the second is the number of iterations.
+    kwargs : dict, optional
+        Additional keyword arguments to pass to ``root_scalar`` such as ``maxiter_ls``,
+        ``alpha``.
+
+    Returns
+    -------
+    coords : ndarray
+        Shape (k, 3).
+        DESC computational coordinates [ρ, θ, ζ].
+    info : tuple
+        2 element tuple containing residuals and number of iterations for each point.
+        Only returned if ``full_output`` is True.
+
+    """
+    kwargs.setdefault("maxiter", maxiter)
+    kwargs.setdefault("tol", tol)
+
+    rho, alpha, zeta = clebsch_coords.T
+    alpha = alpha % (2 * np.pi)
```

Comment:
do we want to modulo alpha by 2pi? I would think for most cases yes but would there ever be a case when we don't want this?

I dont have an example off the top of my head, just want to pose the question

Reply:
We can either do it once here or move the modulo on line 385 to act on `r`. I chose the former